west.yml

```diff
@@ -203,7 +203,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: b5f6470ec7ccd7f1c9a2c929fa93eae9fb001d9e
+      revision: pull/497/head
```

Comment:
manifest update must come before anything needing it

Reply:
yes, will make it the first patch. Thanks